dags/addons.py

```diff
@@ -13,7 +13,7 @@
     'retry_delay': timedelta(minutes=30),
 }
 
-dag = DAG('addons', default_args=default_args, schedule_interval='@daily')
+dag = DAG('addons', default_args=default_args, schedule_interval='@daily', max_active_runs=5)
```

Comment:
The value of this parameter is the same as the default set in the file above, do we really need to enforce it here as well?